recipe/0006-pycryptosat-win32-remove-runtime_library_dirs.patch

```diff
@@ -0,0 +1,23 @@
+--- python/setup.py.in
++++ python/setup.py.in
+@@ -129,8 +129,7 @@
+     extra_compile_args = cconf + ['-I${PROJECT_SOURCE_DIR}', '-I${PROJECT_BINARY_DIR}/cmsat5-src'],
+     extra_link_args = extra_link_args,
+     language = "c++",
+-    library_dirs=['.', '${PROJECT_BINARY_DIR}/lib'],
+-    runtime_library_dirs=['${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}'],
++    library_dirs=['.', '${PROJECT_BINARY_DIR}/lib', '${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}'],
+     libraries = [libname]
+ )
+ 
+--- python/CMakeLists.txt
++++ python/CMakeLists.txt
+@@ -54,7 +54,7 @@
+ 
+ add_custom_command(OUTPUT ${OUTPUT}/timestamp
+                    COMMAND ${PYTHON_EXECUTABLE}
+-                   ARGS setup.py build_ext --inplace --rpath ../lib
```

Comment:
@msoos: `runtime_library_dirs` is not supported by `distutils` for MSVC, so we have to remove it.
Instead of removing the line in `setup.py.in` completely, we can also do something like
```python
    runtime_library_dirs=([] if platform.system() == 'Windows' else ['${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}']),
```
such that the patch can be applied for all OSes. Plus, if we pass on `--rpath` here, `runtime_library_dirs` gets set implicitly. Hence we have to remove that argument, too.
But I also wonder if `--rpath ../lib` generally makes sense or if this should rather be removed altogether either way..